dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 dash_core_components==0.33.0
-dash_html_components==0.11.0rc5
-dash==0.32.0
+dash_html_components==0.13.4
```

Comment:
There's a fix in html comps for react 16, needs to be present for all tests to pass